src/common/components/Select.tsx

```diff
@@ -41,6 +41,20 @@ interface SelectProps {
   value?: SingleValue<SelectOption> | MultiValue<SelectOption>;
 }
 
+export const handleChangeFactory = (
```

Comment:
It might be worth coming up with a prefix we can use for exports which are only meant to be consumed by tests. The pythonista in my thinks `export const _handleChangeFactory` could work well.